customTests/azure_nccl_allreduce.nhc

```diff
@@ -2,8 +2,13 @@
 
 # Check for NVlink ssues by running NCCL allreduce.
 
+TOPOFILE=$3
+if [[ -z "$TOPOFILE" ]];then
```

Comment:
any reason we are using NDv4 topo by default? It maybe better to fail the test if topology file is not passed in.

Reply:
That was an arbitrary decision. @rajvi please have the test fail instead.